libkbfs/bserver_remote.go

```diff
@@ -141,11 +248,45 @@ func (b *blockServerRemoteClientHandler) ShouldRetryOnConnect(err error) bool {
 
 var _ rpc.ConnectionHandler = (*blockServerRemoteClientHandler)(nil)
 
+func (b *blockServerRemoteClientHandler) pingOnce(ctx context.Context) {
+	_, err := b.getClient().BlockPing(ctx)
```

Comment:
What about the `putClient()`? It's conceivable that the `getClient` will stay refreshed but the `putClient` will time out.